serveradmin/serverdb/migrations/0013_change_index.py

```diff
@@ -0,0 +1,25 @@
+# Generated by Django 3.2.16 on 2022-11-03 16:12
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('serverdb', '0012_migrate_change_tables'),
+    ]
+
+    operations = [
+        # Apparently Django doesn't support creating indexes on jsonb keys yet
+        migrations.RunSQL(
+            """
+            CREATE INDEX
+                serverdb_change_change_json_hostname
+            ON
+                serverdb_change (
+                    (change_json->'hostname'), 
+                    (change_type in ('create', 'delete'))
+                );
```

Comment:
I think this can be changed to a real partitial index, as most entries can be excluded and don't need a index entry at all.
So having a `WHERE (change_type in ('create', 'delete'))` and not having the condition as index field. Then it would be still used for all 99,x% of never used entries.

Reply:
Doesn't seem to make a big difference in Query time but the partial index you suggested is much smaller, in our setup 2GB vs 100MB.

Adjusted - Thanks.